float64[] joint_positions
---
bool success
string message
